lib/elixir/pages/references/syntax-reference.md

```diff
@@ -60,7 +60,14 @@ Strings are always represented as themselves in the AST.
 
 ### Charlists
 
-Charlists in Elixir are written in single-quotes, such as `'foo'`. Any single-quote inside the string must be escaped with `\ `. Charlists are made of non-negative integers, where each integer represents a Unicode code point.
```

Comment:
Because this is talking about the syntax, we should mention the single-quotes version, because that's the syntax construct. The sigil construct is mentioned elsewhere. So we should either add a mention that this construct is deprecated or remove this section, but we should not rewrite it to sigils. Does that make sense? :)

Reply:
Got it. Is the updated version good?